app/routes/index.tsx

```diff
@@ -14,6 +16,9 @@ export async function loader({context}) {
   return await context.storefront.query(COLLECTIONS_QUERY);
 }
 
+// @ts-ignore
```

Comment:
why `@ts-ignore`

Reply:
I forgot to ask about that, but it was a part of Tony's solution to the incompatibility between Storybook and Remix's `<Link />`. Just tried giving it a type but it seems like any other workaround would be way too cumbersome.